airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/TyperDeduperUtil.kt

```diff
@@ -0,0 +1,21 @@
+package io.airbyte.integrations.base.destination.typing_deduping
+
+import com.google.common.collect.Streams
+import java.util.*
+
+/**
+ * Extracts all the "raw" and "final" schemas identified in the [parsedCatalog] and ensures they
+ * exist in the Destination Database.
+ */
+fun <T> prepareAllSchemas(parsedCatalog: ParsedCatalog, sqlGenerator: SqlGenerator<T>, destinationHandler: DestinationHandler<T>) {
+    val rawSchema =
+        parsedCatalog.streams.stream().map { stream: StreamConfig -> stream.id.rawNamespace }
+    val finalSchema =
+        parsedCatalog.streams.stream().map { stream: StreamConfig -> stream.id.finalNamespace }
+    val createAllSchemasSql = Streams.concat(rawSchema, finalSchema)
+        .filter { obj: String? -> Objects.nonNull(obj) }
+        .distinct()
+        .map { schema: String? -> sqlGenerator.createSchema(schema) }
+        .toList()
```

Comment:
You should be able to do this entirely in kotlin without bringing in external dependencies
```kotlin
val rawSchema = parsedCatalog.streams.mapNotNull { it.id.rawNamespace }
val finalSchema = parsedCatalog.streams.mapNotNull { it.id.finalNamespace }
val createAllSchemasSql = rawSchema.union(finalSchema)
    .map { sqlGenerator.createSchema(it) }
    .toList()
```